packages/shared/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "@segment/analytics-rn-shared",
+  "version": "1.0.0",
+  "private": true,
+  "description": "Shared dev code between Analytics React Native projects",
+  "main": "index.js",
```

Comment:
Right now it is just shared mocks and config files. 
Mocks get injected through the jest.config so that we don't have to repeat mocking RN and other deps everywhere.
And configs just get extended or referenced by path, hence why it isn't really a package to build right now.